deeppavlov/core/commands/utils.py

```diff
@@ -9,19 +9,17 @@ def set_usr_dir(config_path: str, usr_dir_name='USR_DIR') -> PurePath:
     Make a serialization user dir.
     """
     config = read_json(config_path)
-    try:
+    if 'usr_dir' in config:
         usr_dir = Path(config['usr_dir'])
-    except KeyError:
-        parent = Path(config_path).resolve().parent
-        usr_dir = parent.joinpath(usr_dir_name)
+    else:
+        usr_dir = Path(config_path).expanduser().absolute().parent / usr_dir_name
```

Comment:
Currently usr_dir should be defined relatively to config path, easier to perform experiments and debugging.